src/components/Data/Indexes/ModalCreate.vue

```diff
@@ -0,0 +1,111 @@
+<template>
```

Comment:
To make this component usable in different places. And i think it will be easier to maintain & test or transform into a single page.